srcpkgs/nyxt/template

```diff
@@ -0,0 +1,35 @@
+# Template file for 'nyxt'
+pkgname=nyxt
+version=2.0.0
+revision=1
+wrksrc=nyxt
+build_style=gnu-makefile
+hostmakedepends="sbcl curl pkg-config git"
+makedepends="webkit2gtk-devel libfixposix-devel openssl-devel libgirepository-devel"
+depends="dbus sqlite xclip libfixposix-devel openssl-devel libgirepository-devel"
+short_desc="Keyboard-oriented, extensible web-browser"
+maintainer="0x0f0f0f <sudo-woodo3@protonmail.com>"
+license="BSD-3-Clause"
+homepage="https://nyxt.atlas.engineer/"
+distfiles="https://nyxt.atlas.engineer/static/release/nyxt.tar.gz"
+checksum=71154bc6ef88f176f3f7783e1c0ecc191b6dcdd996e1c67f2bbc417b651e0355
+nostrip=yes
+nopie=yes
+nocross="https://travis-ci.org/void-linux/void-packages/builds/590346382"
```

Comment:
I guess not overriding do_build would make it compile.
Link already expired, please push without nocross so we can see logs.

Reply:
do_build is required because the first target in Makefile is help, unfortunately.